### PR TITLE
Consolidation of the programming exam info resolves #146

### LINKED ISSUE #146: Consolidate and Elucidate Programming Exam Questions
## Overview
In the 26 Sept 2024 TA meeting, we discussed the programming exam and decided to make as many adjustments as possible to:
1. Simplify the test taking experience, AND
2. Support linux users

This work improves upon the work brought up by #142. It relates to the work completed in #136.

## Description
I believe this work is already under way.

Specifically, we want to:
- [x] Combine all Programming Exam information into the [programming exam review](https://github.com/softwareconstruction240/softwareconstruction/blob/main/instruction/programming-exam-review/programming-exam-review.md) page
- [ ] Include instructions for Linux users to attempt to take the exam, respecting the following priorities
    - Prefer to find or borrow a Windows/mac device from a friend
    - Then attempt to use a browser spoofer to take the exam
    - Then reach out to a professor if nothing else will work (there are other options)
- [ ] Remove unnecessary pages from the course blueprint now that the content has been moved into the GitHub page

## Assignment
@Truthwatcher64 Has taken the responsibility for writing the Programming Exam with Linux instructions and combining the information into the GitHub page. He would be marked as the "assignee" of this PR, except that he is not available as an option in this `softwareconstruction` repo.

## Authorization
This was authorized by @leesjensen and @jerodw

### PR DESCRIPTION
**Pull request by Truthwatcher64**
This brings all the available info for the programming exam to github.

It also adds a section for using Linux for the exam

resolves #146

### CODE REVIEW

**Comment by frozenfrank** [comment]
@Truthwatcher64 Can you update the PR description to specifically say "resolves #146"? That special keyword links the this PR to the Issue so that they are closed together. Unfortunately, I don't have permission to make that edit myself, and the special keyword is only effective if it's in the PR description (not in a comment like this).

**Comment by frozenfrank** [comment]
@Truthwatcher64 I see that you changed the title of the PR, but can you change the description?

Screenshot 2024-10-19 at 12 24 45 PM

File: instruction/programming-exam/programming-exam.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+Before taking the exam you should complete all of the steps in the chess phase-0 [getting started](https://github.com/softwareconstruction240/softwareconstruction/blob/main/chess/0-chess-moves/getting-started.md) document, except you will not put the project in a GitHub repo. Just create a folder and set it up with the starter code, and open it in Intellij. This will mean you will have an Intellij project with the provided starter code. You should make sure the project is able to compile and the tests run (but still all fail) before starting. Do not add any code, psuedocode, or notes to your project anywhere, this will be treated as cheating.
+
+You should also open the Javadoc in a web browser tab and the [chess phase-0 specification](https://github.com/softwareconstruction240/softwareconstruction/blob/main/chess/0-chess-moves/chess-moves.md) in another browser tab. If you are taking the test during TA hours you can also have the help queue open. If you need to talk with a TA you can open Zoom for that purpose. All other programs should be closed so that you have only the approved programs open on your computer during the exam. .
```
Comment (leesjensen) [inline instruction/programming-exam/programming-exam.md:78]: Extra space at the end of the paragraph

File: instruction/programming-exam/programming-exam.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+## Warnings and Other Information
+
+**Please read this section carefully**
+
+Once you start the exam, do not close any web browser tabs. Sometimes closing a tab causes Examity to terminate the exam.
+
+You may take the exam at any time during the exam period. Just make sure you give yourself enough time to complete the exam by the end of the exam period.
+
+⚠ The exam is not open until midnight the day of the exam. It closes at 5, so make sure that you start with enough time.
```
Comment (leesjensen) [inline instruction/programming-exam/programming-exam.md:101]: Having a different ending time from all the other assignments has caused significant confusion and frustration for many learners. We are considering changing this to midnight and this reference will create a duplication of the deadline that might be incorrect.

File: instruction/programming-exam/programming-exam.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+You should also open the Javadoc in a web browser tab and the [chess phase-0 specification](https://github.com/softwareconstruction240/softwareconstruction/blob/main/chess/0-chess-moves/chess-moves.md) in another browser tab. If you are taking the test during TA hours you can also have the help queue open. If you need to talk with a TA you can open Zoom for that purpose. All other programs should be closed so that you have only the approved programs open on your computer during the exam. .
+
+You may not open other tabs or access other information during the exam. This is designed to truly test that you have the capability to program the project.
+
+If you are going to make handwritten notes while working have a blank sheet of paper ready to show to the camera at the start of the exam.
```
Comment (leesjensen) [inline instruction/programming-exam/programming-exam.md:82]: Do we need this? Can't they just take notes in a markdown file?

File: instruction/programming-exam/programming-exam.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
+# Programming Exam
+
+🖥️ [Lecture Video](#videos)
+
+This programming exam is a re-creation of your [chess phase-0](https://github.com/softwareconstruction240/softwareconstruction/blob/main/chess/0-chess-moves/chess-moves.md) assignment, but under time constraints and without TA or other support. No notes or resources are allowed except those described here.
```
Comment (leesjensen) [inline instruction/programming-exam/programming-exam.md:5]: "No notes or other resources ..." Sets up the expectation that there are some type of "notes" that are allowed. I think we are getting rid of that possibility.

**Comment by leesjensen** [comment]
This is really good and has great information in it. It is really good to get this all in one place and make it easy to maintain and update and so I am excited to get it into GitHub. It is also something that is really important to get right. Perhaps we should get multiple reviewers on this before we finalize it.

**Review by frozenfrank — CHANGES_REQUESTED** [review]
Please give public access to your fork (or directly give me access) so that I can propose other adjustments to the verbiage and formatting of the document

**Comment by leesjensen** [comment]
@Truthwatcher64 - Now that we have passed the last retake, I think we can merge this. Do you want to resolve @frozenfrank 's comments? Also this seems to be a dead page. There is a link to @jerodw's schedule, but I believe we want to make this part of the core instruction. Can we add a link from the [modules page](https://github.com/softwareconstruction240/softwareconstruction/blob/main/instruction/modules.md)?

**Pull request merged**